indy_client/test/cli/test_send_get_nym_validation.py

```diff
@@ -1,153 +0,0 @@
-from binascii import hexlify
-
-import pytest
-from plenum.common.util import friendlyToRaw
-
-from indy_client.test.cli.constants import INVALID_SYNTAX
-from indy_client.test.cli.helper import createUuidIdentifier, addNym, \
-    createHalfKeyIdentifierAndAbbrevVerkey, createCryptonym
-from indy_common.roles import Roles
-from indy_node.test.helper import check_str_is_base58_compatible
-
-CURRENT_VERKEY_FOR_NYM = 'Current verkey for NYM {dest} is {verkey}'
-CURRENT_VERKEY_FOR_NYM_WITH_ROLE = 'Current verkey for NYM {dest} is ' \
-                                   '{verkey} with role {role}'
-CURRENT_VERKEY_IS_SAME_AS_IDENTIFIER = \
-    'Current verkey is same as identifier {dest}'
-NYM_NOT_FOUND = 'NYM {dest} not found'
-
-
-def testSendGetNymSucceedsForExistingUuidDest(
```

Comment:
This tests need to be migrated to SDK and moved to indy-node folder

Reply:
Restored. Will be integrated in future PRs